src/IECoreScene/MeshAlgoFaceArea.cpp

```diff
@@ -97,10 +97,10 @@ PrimitiveVariable MeshAlgo::calculateFaceArea( const MeshPrimitive *mesh, const
 PrimitiveVariable MeshAlgo::calculateFaceTextureArea( const MeshPrimitive *mesh, const std::string &uvSet, const std::string &position, const Canceller *canceller )
 {
 	PrimitiveVariable::Interpolation uvInterpolation = PrimitiveVariable::Vertex;
-	ConstV2fVectorDataPtr uvData = mesh->expandedVariableData<V2fVectorData>( uvSet, PrimitiveVariable::Vertex );
+	ConstV2fVectorDataPtr uvData = mesh->expandedVariableData<V2fVectorData>( uvSet, PrimitiveVariable::Vertex, false, canceller );
 	if( !uvData )
 	{
-		uvData = mesh->expandedVariableData<V2fVectorData>( uvSet, PrimitiveVariable::FaceVarying );
+		uvData = mesh->expandedVariableData<V2fVectorData>( uvSet, PrimitiveVariable::FaceVarying, false, canceller );
```

Comment:
It seems like we should really be using an `IndexedView` here rather than generating a whole load of data we're just going to throw away?

Reply:
Addressed in separate commit.